portal-web/test/functional/com/liferay/portalweb/macros/analyticscloud/ACIndividualsDashboard.macro

```diff
@@ -14,6 +20,33 @@ definition {
 		Click(locator1 = "ACIndividualsDashboard#DELETE_BREAKDOWN_BUTTON");
 	}
 
+	macro goToDistributionTab {
+		ACIndividualsDashboard.goToTabName(tabName = "Distribution");
+	}
+
+	macro goToInterestsTab {
+		ACIndividualsDashboard.goToTabName(tabName = "Interests");
+	}
+
+	macro goToKnownIndividualDetails {
+		var key_individualName = "${individualName}";
+		Click(locator1 = "ACIndividualsDashboard#KNOWN_INDIVIDUALS_LIST_ITEMS");
+	}
+
+	macro goToKnownIndividualsTab {
+		ACIndividualsDashboard.goToTabName(tabName = "Known Individuals");
+	}
+
+	macro goToOverviewTab {
+		ACIndividualsDashboard.goToTabName(tabName = "Overview");
+	}
+
+	macro goToTabName {
```

Comment:
Vinicius was working on a navigation macro that goes to different tabs that should function similarly to this. It's currently not in upstream though but might be better to use the Vinicius macro when it's ready. https://github.com/alexcandidos/liferay-portal/pull/445/files#diff-d5db08b0debdc60351336dafb3ed0f6ad65e4d11fd749b06ef0e3b0aacf1fb80.